playwright/Pipfile

```diff
@@ -0,0 +1,20 @@
+[[source]]
+verify_ssl = true
+url = "https://pypi.org/simple"
+name = "pypi"
+
+[packages]
+# general
+"Fabric3" = "*"                             # task automation
+tqdm = "*"                                  # progress bar in dev fab tasks
+requests = "*"
+pyquery = "*"
+pytest = "*" 
+playwright = ">=1.18"
+pytest-playwright = "*"
+
+[requires]
+python_version = "3.8"
+
+[pipenv]
+allow_prereleases = true
```

Comment:
Do you happen to know if we need pre-releases here? Perma's main env had to add that to solve a particular problem, but it sometimes bites us, switching us to alpha packages that are not ready for primetime. Might be nice to -not- include, unless we do in fact need it.

Reply:
Confirmed that we do not. I was cleaving close to the perm_web setup, and we can remove this.